dimtu/variables.tf

```diff
@@ -0,0 +1,53 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Description of arguments can be found in
+# ../modules/single-machine/variables.tf in this repository
+# ----------------------------------------------------------------------------------------------------------------------
+
+variable "flavor" {
+  default = "m3.small"
```

Comment:
Could we up this to `m3.quad`? This machine is likely to acquire substantially more containers over time.